parser-java/label-detection-web-app/README.md

```diff
@@ -0,0 +1,52 @@
+# KVS Label Detection Web Application Sample
```

Comment:
Could you also add some snippets of your web app? Also, maybe add them in the flow that you expect them to follow. This will be helpful to guide your users.